iogt/templates/core/tags/section-and-subsection_listing_homepage.html

```diff
@@ -0,0 +1,87 @@
+{% load wagtailcore_tags wagtailimages_tags comments static core_tags i18n %}
+
+{% if sections %}
+<h3 class="heading heading--homepage">{% trans "Recommended" %}</h3>
+<div class="section-list section-list__homepage">
```

Comment:
This should be a modifier of the original, so it should be `section-list--homepage`